ui/imports/shared/status/StatusSettingsLineButton.qml

```diff
@@ -113,13 +124,19 @@ Rectangle {
 
     MouseArea {
         anchors.fill: parent
-        enabled: root.isEnabled
```

Comment:
I think we should keep this line.
I suppose you made this to keep the cursorShape, but I think it shouldn't be done. `PointingHandCursor` should only appear if it's possible to make some action.

Reply:
And `StatusToolTip` could be not a child of `MouseArea`, then it will stay anaffected.

Reply:
hum, the `enabled` is removed because we need to make the MouseArea communicate the onEntered end onExited when the Switch is disabled.